modules/monitoring/metrics_discovery.py

```
@@ -373,14 +373,98 @@ def _resolve_unit_utilization(unit_type: str, unit_id: Any) -> int | None:
+    try:
+        res = process_exec.check_output_text(_resolve_amd_smi_cmd(smi_bin), timeout=5.0)
+        return _parse_amd_smi_csv(res)
+    except (process_exec.CommandExecutionError, process_exec.CommandTimeoutError, FileNotFoundError, ValueError):
+        return []
+
+
+def _amd_sysfs_paths(idx: int) -> list[str]:
+    return [
+        f"/sys/class/drm/card{idx}/device/gpu_busy_percent",
+        f"/sys/class/drm/renderD{128 + idx}/device/gpu_busy_percent",
+    ] + glob.glob("/sys/class/drm/card*/device/gpu_busy_percent")
```

Ruff (0.16.1)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)